README.md

```diff
@@ -1,4 +1,88 @@
+  <h3><b>Hello Microverse Project</b></h3>
 
+# 📗 Table of Contents
 
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
+# 📖 [Hello Microverse] <a name="about-project"></a>
 
+**[Hello Microverse]** is the first project in the first block of the first module of Microverse full stack web development program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/H-Ramalan)
+- Twitter: [@twitterhandle](https://twitter.com/hamza_ramalan)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please you can support me by starring it on github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partner, my morning session team and also my stand up team for being very supportive during this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[What is the project about?]**
+
+  - [It is my first project at microverse]
+
+- **[will there be additional features?]**
+
+  - [Unfortunately, it is a simple project with very little chance of having any added features in the future.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is  [MIT license](https://choosealicense.com/licenses/mit/)(./LICENSE) licensed.
+
```

Comment:
- Kindly Make Sure the Link to the License File redirects to the LICENCE in your project. I would recommend you to remove this line `(https://choosealicense.com/licenses/mit/)` to make it work.